sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientTest.java

```diff
@@ -504,12 +551,19 @@ void updateQueueWithResponse() throws IOException {
                 }
 
                 final CreateQueueBody argument = (CreateQueueBody) arg;
-                return argument.getContent() != null && argument.getContent().getQueueDescription() != null;
+                if (argument.getContent() == null || argument.getContent().getQueueDescription() == null) {
+                    return false;
+                }
+                assertEquals(argument.getContent().getQueueDescription().getForwardTo(), FORWARD_TO_ENTITY,
```

Comment:
iirc the order of parameters is `expected, actual, message`

Reply:
ah nice catch, rewriting the code over and over again leads to all kinds of weirdness :)
fixed